reana_workflow_controller/utils.py

```diff
@@ -10,11 +10,12 @@
 import os
 from pathlib import Path
 
-import fs
-from flask import current_app as app
 from reana_db.database import Session
 from reana_db.models import Job, JobCache
 
+import fs
+from flask import current_app as app
+from reana_commons.config import REANA_WORKFLOW_UMASK
```

Comment:
Please update `setup.py` to require the newest `reana-commons` release for the import to work